src/Models/EntityRelation.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace KusikusicmsModels\Models;
+
+use Illuminate\Database\Eloquent\Relations\Pivot;
+use Illuminate\Database\Eloquent\Model;
+
+class EntityRelation extends Pivot
+{
+    const RELATION_ANCESTOR = 'ancestor';
+    const RELATION_MEDIA = 'medium';
+    const RELATION_UNDEFINED = 'relation';
+    const RELATION_MENU = 'menu';
```

Comment:
I will try to define here just ancestor and undefined, because the other kinds or relations belong to other packages